http/http_client.go

```diff
@@ -131,6 +132,44 @@ func (c *HttpClient) getActualDomain(ctx context.Context) string {
 }
 
 func (c *HttpClient) doRequest(ctx context.Context, req *http.Request, headers map[string][]string, midList []ReqMiddleWare) ([]byte, map[string]interface{}, error) {
+	quota := utils.GetPodRateLimitQuotaFromCtx(ctx)
+	oldQuota := limiter.maxRequest
+	if reset := limiter.ResetRateLimiter(quota); reset {
+		fmt.Println(fmt.Sprintf("rate limit reset from %d to %d, apiID: %s", oldQuota, quota, utils.GetFuncAPINameFromCtx(ctx)))
+	}
+
+	if !limiter.AllowRequest() {
+		format := "request limit exceeded quota: %d qps"
+		formatLog := utils.FormatLog{
+			Level:         utils.LogLevelWarn,
+			EventID:       utils.GetExecuteIDFromCtx(ctx),
+			FunctionAPIID: utils.GetFunctionAPIIDFromCtx(ctx),
+			LogID:         utils.GetLogIDFromCtx(ctx),
+			Timestamp:     time.Now().UnixNano() / 1e3, // 使用微秒
+			Message:       fmt.Sprintf(format, quota),
+			TenantID:      utils.GetTenantIDFromCtx(ctx),
+			TenantType:    utils.GetTenantTypeFromCtx(ctx),
+			Namespace:     utils.GetNamespaceFromCtx(ctx),
+			LogType:       constants.RateLimitLogType,
+		}
+
+		if c.rateLimitLogCount < utils.LogCountLimit {
+			c.rateLimitLogCount++
+			content := utils.GetFormatLogWithMessage(formatLog, c.rateLimitLogCount)
```

Comment:
此时 c.rateLimitLogCount == 10000 时，消息会被加上后缀：“The log has been discarded because it exceeded the limit of  10000”，并被打印出来。

可以看下，是不是把 GetFormatLogWithMessage 函数中 if streamLogCount == LogCountLimit { 改成 if streamLogCount > LogCountLimit {